andela/kingsmen-ah-backend
# #166272333 jwt successful registration

#### What does this PR do?
- Successfully generates a token on successful login and registration
#### Description of Task to be completed?
- Write a function to create a token from the payload supplied.
- Generate token when user signs in or register.
- Implement login functionality.
- Reroute the current endpoint to make use of `POST api/v1/auth/login.js`.
- Create a function to verify a token.
#### How should this be manually tested?
- Navigate to the URL: `POST /api/v1/auth/register` then, register a new user.
- Navigate to the URL: `POST /api/v1/auth/login` and login a user. Verify the token gets displayed.
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#166272333](https://www.pivotaltracker.com/story/show/166272333)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
